etna/ensembles/voting_ensemble.py

```diff
@@ -116,20 +116,12 @@ def _vote(self, forecasts: List[TSDataset]) -> TSDataset:
         forecast_dataset = TSDataset(df=forecast_df, freq=forecasts[0].freq)
         return forecast_dataset
 
-    def forecast(self, prediction_interval: bool = False) -> TSDataset:
-        """Forecast with ensemble: compute weighted average of pipelines' forecasts.
-
-        Parameters
-        ----------
-        prediction_interval:
-            This parameter is ignored
-
-        Returns
-        -------
-        TSDataset:
-            dataset with forecasts
+    def _forecast(self) -> TSDataset:
+        """Make predictions.
+        Compute weighted average of pipelines' forecasts
         """
-        self.check_support_prediction_interval(prediction_interval)
+        if self.ts is None:
+            raise ValueError("Something went wrong, ts is None inside the _forecast!")
```

Comment:
We do not need to specify ` inside the _forecast!` bit, because we will have stack trace.
```suggestion
            raise ValueError("Something went wrong, ts is None")
```

Reply:
Done